.github/workflows/network.yml

```diff
@@ -0,0 +1,70 @@
+name: Network CI
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'lib/vdsm/network/**'
+      - 'lib/vdsm/common/network/**'
+      - 'tests/network/**'
+      - '.github/workflows/network.yml'
+  pull_request:
+    paths:
+      - 'lib/vdsm/network/**'
+      - 'lib/vdsm/common/network/**'
+      - 'tests/network/**'
+      - '.github/workflows/network.yml'
+
+jobs:
+  unit:
+    env:
+      IMAGE_TAG: ${{ matrix.tag }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tag: [ centos-8, centos-9 ]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+          sudo apt update
+          sudo apt install podman
+      - name: Run unit tests
+        run: sudo -E ./tests/network/unit/run-tests.sh
```

Comment:
Why do you need to use podman? It does not work with the builtin docker support?

Reply:
I had some issues with docker and systemd in containers. Podman is proven to work with that out of the box.